modules/nw-ovn-k-adminnetwork-policy-action-rules.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: <Concept>
+[id="AdminNetworkPolicy-actions-for-rules"_{context}]
```

Comment:
skipping this file because I think its  mostly a copy from blogpost https://www.redhat.com/en/blog/using-adminnetworkpolicy-api-to-secure-openshift-cluster-networking and me reviewing this is conflict of interest...perhaps peer review is enough here.. We might want to link to that blogpost as a resource ?